dist/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Big Chefs</title>
+    <script src="https://kit.fontawesome.com/84967187a9.js" crossorigin="anonymous"></script>
+<script defer src="index.bundle.js"></script></head>
+<body>
+    <header><span>Big Chief</span><span id="bars" class="fas fa-bars"></span></header>
+    <nav>
+        <div class="logo"><!-- Logo from index.js --></div>
+        <div class="menu">
+            <ul>
+                <li><a href="#Home">Home</a></li>
+                <li><a href="#about">about</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </div>
+    </nav>
+    <main>
+        <section class="main_head">
+            <h1>
+                <span></span>
+                Big<br>Chief
+            </h1>
+            <p>Find and share everyday cooking inspiration on big Chief. Discover recipes, cooks, videos, and how-tos based on the food you love.</p>
+        </section>
+        <section class="meals_container">
+            <!-- Articles from got from www.themealdb.com/api.php -->
+        </section>
+        <section class="comment-section">
+            <div class="close-icon"></div>
+            <div class ="meal-display">
+            <div class ="post-info"></div>
+        </div>
+            <div class="display-comments">
+                <h2>Comments  <span class="comment-head"></span></h2>
+            </div>
+        
+            <div>
+            <ul class ="comment-list">
+                    
+            </ul>
+        </div>
+            <form class="comment">
+                <h2 class ="add-comment">Add a comment</h2>
+                 <input type="text" name="name" id="name" placeholder="your name" class="comment-input" required><br>
+                 <textarea id="txtid" name="txtname" class="comment-area" rows="3" cols ="20" required>Write a comment</textarea><br>
+                <button class="comment-place-btn" type="submit">Comment</button>
```

Comment:
When the Comment button is clicked, the user is redirected to the home page. What should happen is that the comment should be displayed on the UI and the data should be recorded by the Involvement API.

- [ ] Please, fix this issue, in order that after the button is clicked, the user remains on that particular recipe, with it's comment recorded and updated on the UI. :+1: